modules/flowable-osgi/src/test/java/org/flowable/osgi/BundleScriptEngineResolverTest.java

```diff
@@ -0,0 +1,55 @@
+package org.flowable.osgi;
```

Comment:
Please include the standard license header file; thank you.

Reply:
Sure, the license header has been added.